### DISCUSSION TITLE
what are these numbers?

### DISCUSSION THREAD

**Discussion by itsmmt305** [libre-tube/LibreTube#3476] [Q&A] [answered]
![Screenshot_2023-04-02-10-34-43-491_com github libretube](https://user-images.githubusercontent.com/94820934/229339214-4e235d10-2360-4fc3-b722-2e851d6d07b1.jpg)

12, 18, 19, 61 etc.. i checked with the view counts and the published dates. no matches.

**Comment by Bnyro** [comment, marked as answer]
Good question, looks like there's an issue with the formatting of the view count!
